src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json

```diff
@@ -22,7 +22,7 @@
 			{
 				"Sequence": 1,
 				"Name": "library",
-				"Type": "tooltip",
+				"Type": "library",
```

Comment:
@QilongTang @RobertGlobant20 @filipeotero  - this seems really fragile to me, if we change the name of a control then the guides tour will break and we won't know - are there tests to ensure these names don't change without the json dat a being updated?

Reply:
@mjkkirschner I agree that is fragile, I will create a Jira task for adding some unit tests that will be validating that the names in the dynamo_guides.json exists in the Dynamo VisualTree. Thanks

Reply:
@mjkkirschner Yup, we discussed this issue but have not finalized the right strategy for unit testing this. We can read this Json object for sure, do you think we should search for such control under DynamoView tree and look for a matching?